data_model/src/transaction.rs

```diff
@@ -250,11 +252,12 @@ impl SignedTransaction {
         &tx.payload.metadata
     }
 
-    /// Creation timestamp as [`core::time::Duration`]
+    /// Creation timestamp
     #[inline]
-    pub fn creation_time(&self) -> Duration {
+    #[cfg(feature = "std")]
+    pub fn creation_time(&self) -> SystemTime {
```

Comment:
it's needed

Reply:
As a proposal, we could use [`jiff::Timestamp`](https://docs.rs/jiff/latest/jiff/struct.Timestamp.html)/[`chrono::DateTime`](https://docs.rs/chrono/latest/chrono/struct.DateTime.html) as a main type instead of `std::time::SystemTime`. Both can work in no-std context.

Reply:
I think `core::time::Duration` was a painstaking effort to make it compatible with no_std. Is it worth having separate methods to achieve semantic consistency? A time span from the origin can be considered a timestamp

Reply:
We need to decide something. Either introducing other `no_std` compatible types or staying in `core::time::Duration` is acceptable to me